package.json

```diff
@@ -21,7 +21,7 @@
         "test-back": "yarn back pytest",
         "test-func": "yarn func pytest",
         "test": "yarn test-back && yarn test-front && yarn test-func",
-        "start-front": "yarn fyarn stop && yarn front yarn serve",
+        "start-front": "yarn front yarn serve",
```

Comment:
This is not really necessary, but it makes my life as a Windows user a lot easier. `fyarn stop` uses `lsof`, a Linux-only command.